### PR TITLE
Js/user layout

### PR DESCRIPTION
**Pull request by joshstricklin**
Add basic navigation.

Show working "Log in" link if logged out.

Show working "Log out" link if logged in.

### CODE REVIEW

**Comment by joshstricklin** [comment]
[ch32]
[ch33]
[ch37]
[ch38]

**Comment by joshstricklin** [comment]
[ch37]
[ch38]

File: app/views/application/_header.slim [hunk 1]
```diff
@@ -0,0 +1,9 @@
+header
+	h1 = link_to "Taskopolis", root_path
+	nav
+		ul
+			- if user_signed_in?
+				li = link_to "Log out", destroy_user_session_path, method: :delete
+			- else
+				li = link_to "Register", new_user_registration_path
+				li = link_to "Log in", new_user_session_path
```
Comment (vishalarora) [inline app/views/application/_header.slim:9]: Are you using tabs or spaces?  We're going to get all kinds of merge conflicts if we don't use the same thing.

Any objection to moving to 2 spaces?

File: db/schema.rb [hunk 2]
```diff
@@ -15,6 +15,21 @@
...
+    t.text     "last_error"
+    t.datetime "run_at"
+    t.datetime "locked_at"
+    t.datetime "failed_at"
+    t.string   "locked_by"
+    t.string   "queue"
+    t.datetime "created_at"
+    t.datetime "updated_at"
+    t.index ["priority", "run_at"], name: "delayed_jobs_priority", using: :btree
+  end
```
Comment (vishalarora) [inline db/schema.rb]: You have this in both PRs.  Whenever you merge the #11, you'll want to rebase this before merging.

File: spec/factories.rb [hunk 3]
```diff
@@ -1,8 +1,5 @@
 FactoryGirl.define do
   factory :list do
-  end
-
-  factory :list do
     name "MyString"
```
Comment (vishalarora) [inline spec/factories.rb]: Same comment re rebasing.

**Review by vishalarora — APPROVED** [review]
Some style comments but this LGTM.

File: app/views/application/_header.slim (hunk 1 above)
Comment (joshstricklin) [reply app/views/application/_header.slim:9]: I'm using tabs, but no objections to changing.

No objection to 2 spaces! I'll configure ST to accommodate.

File: db/schema.rb (hunk 2 above)
Comment (joshstricklin) [reply db/schema.rb]: Whoops! Will do.

**Pull request merged**